lib/koans/04_numbers.ex

```diff
@@ -0,0 +1,93 @@
+defmodule Numbers do
+  require Integer
+  use Koans
+
+  @intro "Numbers"
```

Comment:
I really like the titles for your koans and I know we haven't put tooo much effort into the `@intro ....` text. It would be awesome if this koans were a shining example :) Do you mind adding a little text, funny and explanatory about numbers in Elixir? 2-3 lines should suffice.